src/main/java/org/mskcc/oncokb/curation/service/TranscriptService.java

```diff
@@ -582,6 +582,77 @@ public List<EnrichedAlignmentResult> getAlignmentResult(
         }
     }
 
+    public List<ProteinExonDTO> getExons(Gene gene, ReferenceGenome referenceGenome) {
+        Optional<TranscriptDTO> transcriptOptional = this.findByGeneAndReferenceGenomeAndCanonicalIsTrue(gene, referenceGenome);
+        if (transcriptOptional.isPresent()) {
```

Comment:
Nit: I generally prefer to write something like this as `!transcriptOptional.isPresent()` and doing an early return. In my head, I can easily read the disqualifying conditions first (and it also reduces nesting), which means that I read the code in an order that I can follow the logic.

Definitely preference, so if you don't agree, feel free to leave alone